### PR TITLE
add how to use the application

### LINKED ISSUE #8: README
Create README.md, it should contain:

- description of the project
- team members
- language (English or German)
- document the license and the code of conduct
- tell users how to launch the application

### PR DESCRIPTION
**Pull request by EdenTewelde**
closes #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,13 @@ Our lecturer is [Nicolai Parlog](https://github.com/nicolaiparlog).
 
 We have to solve the [tasks](https://olat.vcrp.de/auth/RepositoryEntry/1676804160/CourseNode/95999652079122) in the limited period from 30.07.2017 (18:00) to 13.08.2017 (18:00).
 
+## Application
+To start our application first you have to open command prompt or Windows PowerShell. Navigate to the directory the jar-file is in. There you can run the following command:
+'''
```
Comment (nipafx) [inline README.md]: I assume you're going for a fenced code block here - they need to start with three backticks (not apostrophes):

    ```
    code goes here
    ```

It is also customary (maybe even necessary) to surround the block with empty lines.

Comment (EdenTewelde) [reply README.md]: Yes, that's what I was going for.

Comment (EdenTewelde) [reply README.md]: That's the problem when working with an editor that doesnt support Markdown 😉

**Comment by nipafx** [comment]
:+1:

**Pull request merged**